src/python/pants/backend/shell/target_types.py

```diff
@@ -452,11 +444,13 @@ class ShellCommandWorkdirField(StringField):
     )
 
 
-class RunShellCommandWorkdirField(ShellCommandWorkdirField):
-    default = None
+class RunShellCommandWorkdirField(StringField):
+    alias = "workdir"
+    default = "."
     help = softwrap(
-        "Sets the current working directory of the command that is `run`. If `None`, run the "
-        "command from the directory you are invoking Pants from."
+        "Sets the current working directory of the command that is `run`. Values that begin with "
+        "`.` are relative to the directory you are running Pants from. Values that begin with `/` "
+        "are from the root of your filesystem."
```

Comment:
if `workdir` begin with `/` is it not rooted in the project root? 
Seems like it would be a bad idea to support having paths outside the project in BUILD files..?
Yea, looking further down, iiuc it's rooted in the build root.

Reply:
good catch!